pilot/pkg/features/pilot.go

```diff
@@ -328,4 +328,12 @@ var (
 			"No need to configure this for root certificates issued via Istiod or web-PKI based root certificates. "+
 			"Use || between <trustdomain, endpoint> tuples. Use | as delimiter between trust domain and endpoint in "+
 			"each tuple. For example: foo|https://url/for/foo||bar|https://url/for/bar").Get()
+
+	EnableDefaultToSystemCAValidation = env.RegisterBoolVar("USE_SYSTEM_CA_FALLBACK", false,
+		"Description exists.",
+	)
+
+	SystemCAFilePath = env.RegisterStringVar("SYSTEM_CA_FILE_PATH", "/etc/ssl/certs/ca-certificates.crt",
```

Comment:
"./etc/" - all files must start with ".", so we can run Istiod and Agent without root access on developer (or prod) machines.

But this may be an exception.